x/mobileproxy/README.md

```diff
@@ -0,0 +1,176 @@
+# Mobileproxy: Local Proxy Library for Mobile Apps
+
+This package enables the use Go Mobile to generate a mobile library to run a local proxy and configure your app networking libraries.
+
+### Build the Go Mobile binaries with [`go build`](https://pkg.go.dev/cmd/go#hdr-Compile_packages_and_dependencies)
+
+```bash
+go build -o ./out/ golang.org/x/mobile/cmd/gomobile golang.org/x/mobile/cmd/gobind
+```
+
+### Build the iOS and Android libraries with [`gomobile bind`](https://pkg.go.dev/golang.org/x/mobile/cmd/gomobile#hdr-Build_a_library_for_Android_and_iOS)
+
+```bash
```

Comment:
👍 Providing a wrapper library would be better than documentation.

Reply:
That is definitely something that will help and we should consider. The downside is that it will require a releases process. Let's see how people end up using it and think about where we would put the binaries. To be defined.

Reply:
For now, I don't want to set the expectation that we will be providing the pre-packaged libraries. I want to see if what we have here will be good enough first, and we can consider the next step if people keep struggling.

Reply:
The reason for not having a TODO for the release is that we are not currently intending to do it. Adding a TODO to a README file will set the wrong expectation. It's different from a code TODO.

We've been keeping the roadmap in the root README.